styles.css

```diff
@@ -861,3 +861,58 @@ form small {
     left: -20px;
   }
 }
+
+/* Pop up window styling */
+.fixed_postion {
+  position: fixed;
+  filter: blur(8px);
+}
+
+.hide {
+  display: none;
+}
+
+.fixed_headline {
+  position: fixed;
+  filter: blur(12px);
+}
+
+.see-source {
+  display: flex;
+  width: 135px;
+  margin-right: 20px;
+}
+
+.see-source-img {
+  padding-left: 15px;
+}
+
+.project-detail-work-description {
+  font-size: 14px;
+}
+
+.project-detail-buttons li {
+  margin-right: 10px;
+}
+
+.project-detail-buttons {
+  display: flex;
+}
+
+.project-detail-buttons a {
+  text-decoration: none;
+}
+
+.project-detail-header {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly make your design close to the Figma template as much as possible. You may fix the font color of your text so that your design can match the design provided.
- [ ] Kindly fix your image of the desktop popup so that it doesn't appear shrunk. Make sure your image is a replica of the design provided in the Figma template.
- Your design
![Your desktop popup](https://user-images.githubusercontent.com/63917725/189131648-31d3d8f0-91fd-4254-aaa2-b0a071c70134.PNG)

- Figma design
![Figma popup](https://user-images.githubusercontent.com/63917725/189131727-09eac26d-7a9b-4d4a-8b0f-288a73dc8af2.PNG)

- [ ] You may also fix the margin-bottom of your mobile popup so that the whole mobile popup can appear on the screen.
- [ ] Also fix the font color of the mobile popup so that it matches the design provided in the Figma template.
![Your mobile popup](https://user-images.githubusercontent.com/63917725/189132528-9e4b6389-0d5d-4e00-ad3b-2c4ef35b7e25.PNG)

- [ ] You may kindly fix the overflowing areas of your design so that it can be responsive. Some parts seem to overlap as the screen size changes.
![responsiveness](https://user-images.githubusercontent.com/63917725/189133169-5f99a9f1-a5f7-4def-87b6-342083eb8891.PNG)

Reply:
We have worked on the suggested changes. Thank you.